hack/dump-debug-info.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+source hack/common.sh
+
+echo "dumping debug information"
+echo "--- Service Account ---"
+${OC_PATH} get serviceaccount
```

Comment:
I don't see OC_Path defined anywhere.

Reply:
Removed in an earlier commit. Fixed now.